system_tests/vision.py

```diff
@@ -63,7 +63,6 @@ def _assert_coordinate(self, coordinate):
         if coordinate is None:
             return
         self.assertIsInstance(coordinate, (int, float))
-        self.assertNotEqual(coordinate, 0.0)
```

Comment:
So with a face, there could be a facial feature that starts at coordinate 0,0 of a facial bounds box which would be 0,0.